packages/lint/eslint/eslint-config-typescript/index.js

```diff
@@ -28,9 +28,18 @@ module.exports = {
         '@typescript-eslint/camelcase': 'error',
         '@typescript-eslint/class-name-casing': 'error',
         '@typescript-eslint/explicit-function-return-type': 'warn',
-        '@typescript-eslint/explicit-member-accessibility': 'error',
+        '@typescript-eslint/explicit-member-accessibility': [
+          'error',
+          {
+            accessibility: 'explicit',
+            overrides: {
+              constructors: 'no-public',
```

Comment:
This is to avoid having to write `public constructor(...)` each time, allows to write `constructor(...)`

Reply:
yes i saw the doc, but each time we change something in eslint i end up checking those documentation pages, that's why i asked to include them in the commit. Maybe we could also embed them as comments in the js file above each property ?

Reply:
OK for the commit message. For the comment, I suggest adding https://github.com/typescript-eslint/typescript-eslint/tree/v1.7.0/packages/eslint-plugin/docs/rules (fixed version) at the top of the file. Must be updated when we update the version then.

Reply:
If we integrate https://github.com/kentcdodds/preval.macro we could put a check at build time that the version in the docs is the same as the version in the package.json